src/Platform/Microsoft.Testing.Platform/Services/PoliciesService.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Concurrent;
+
+namespace Microsoft.Testing.Platform.Services;
+
+internal sealed class PoliciesService : IPoliciesService
+{
+    internal sealed class Policy
```

Comment:
I would probably move this type up and have it part of the interface.

Reply:
Even if not used in the interface at all?

Reply:
Class is removed now completely :(